core/src/player.rs

```diff
@@ -373,6 +375,8 @@ impl Player {
         self.instance_counter = 0;
 
         self.mutate_with_update_context(|context| {
+            context.avm2.root_api_version = ApiVersion::from_u8(context.swf.version())
```

Comment:
This is not correct, right? We want SWF v23 to map to `SWF_23`, not `AIR_3_4`.

Reply:
I've fixed this